bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -0,0 +1,1312 @@
+Switch   East_Camera_UpdateGif                       "Create an animated gif"               (gCameraGif)    {channel="ipcamera:HIKVISION:East:updateGif"}
+Switch   Front_Camera_UpdateGif                   "Create an animated gif"                  (gCameraGif)    {channel="ipcamera:HIKVISION:Front:updateGif"}
+Switch   West_Camera_UpdateGif                    "Create an animated gif"                  (gCameraGif)    {channel="ipcamera:HIKVISION:West:updateGif"}
+Switch   Garage_Camera_UpdateGif                  "Create an animated gif"                  (gCameraGif)    {channel="ipcamera:HTTPONLY:Garage:updateGif"}
+Switch   Driveway_Camera_UpdateGif                   "Create an animated gif"               (gCameraGif)    {channel="ipcamera:HTTPONLY:Driveway:updateGif"}
+Switch   LivingRoom_Camera_UpdateGif              "Create an animated gif"                  (gCameraGif)    {channel="ipcamera:HTTPONLY:LivingRoom:updateGif"}
+```
+
+By default all GIFs are saved with `ipcamera.gif` name.
+Now we can archive the old ones by renaming them with a timestamp:
+
+camera.rules:
+
+```java
+var Timer timer = null // top of the file, optionally after imports
+
+rule "Rename GIF Anims when Saved"
+    when
+        Member of gCameraGif changed to ON
+   then
+        val String timeNow = String::format( "%1$tY%1$tm%1$td-%1$tT", new java.util.Date ).replace(":", "")
+        val camera = triggeringItem.name.split("_Camera_").get(0).toLowerCase + "/"
+
+        // Wait 12 secs for ffmpeg to complete
+        if (timer === null) {
+            timer = createTimer(now.plusSeconds(12), [ |
+
+                // Same as FFMPEG_OUTPUT parameter in Thing definition
+                var filePath = "/etc/openhab2/html/cameras/camera-" + camera
+                var oldFileName = (filePath + "ipcamera.gif")
+
+                // Separator between filename and timestamp, e.g. '_', '-', '.', or whatever you prefer
+                var fileDateSeparator = "_"
+
+                // Substitute "ipCam2" to your new filename preference
+                var newFileName = (filePath + "ipcamera" + fileDateSeparator + timeNow + ".gif")
+
+                // Options are "mv" to rename file or "cp" to copy file
+                var methodUsed = "cp"
+
+                executeCommandLine("/bin/" + methodUsed + " " + oldFileName + " "  + newFileName)
+                logInfo(oldFileName, "Copied to " + newFileName)
+            ])
+        } else {
+            if(timer !== null) {
+                timer.cancel
+                timer = null
+            }
+       }
+end
+```
+
+Note that the example above also implies that you use the same naming convention (`[Room]_Camera_[Action]`) and the folders where your GIF files are stored are called `/camera-[room]/` where `[room]` is simply `[Room]` but in lowercase.
+
+
+## Using the LOGS to show DEBUG or TRACE
```

Comment:
Many parts of this chapter applies to all bindings in general. Can you condense this down to what is relevant for this binding and link to the OH documentation for the rest?

Reply:
Removed it to make readme shorter.